RealifeTech/RealifeTech.swift

```diff
@@ -66,6 +67,7 @@ public class RealifeTech {
             graphQLManager: graphQLManager,
             orderingJourneyUrl: configuration.webOrderingJourneyUrl,
             colorStore: General)
+        Identity = IdentityFactory.makeIdentityModule()
     }
 
     public static func clearAllInterfaces() {
```

Comment:
I wonder if we need to clear `Identity` in this function because `Identity` contains a GraphQL mutation which has the bearer token header in it. But let's see if it's necessary when implementing the mutation

Reply:
I've just added it here.